Cargo.toml

```diff
@@ -77,6 +77,7 @@ napi = { version = "2.14.2", features = ["compat-mode", "napi8", "serde-json"] }
 napi-build = { version = "2.1.0" }
 napi-derive = { version = "2.14.6" }
 once_cell = { version = "1.19.0" }
+nom = { version = "7.1.3" }
```

Comment:
nit: I suggest keeping the list alphabetically sorted.

Reply:
Done